content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -224,45 +251,32 @@ If there are errors that you don't understand, or if traces are reported as flus
 {{< /programming-lang >}}
 {{< programming-lang lang="python" >}}
 
-For more visibility, include `DD_TRACE_LOGGING_RATE=0`.
+Logs generated by the Python Tracer have the logging handler name `[ddtrace]`. 
```

Comment:
```suggestion
Logs generated by the Python Tracer have the logging handler name `ddtrace`. 
```

It showing up as `[ddtrace]` is an implementation detail of the log format they are using